server/controllers/notification.js

```diff
@@ -58,30 +90,33 @@ exports.markAsread = expressAsyncHandler(async (req, res, next) => {
     res.status(404).json({
       error: {
         message: "Not found",
-      }
+      },
     });
-    throw new Error('Not found');
+    throw new Error("Not found");
   }
 
   notification.read = true;
   notification.read_date = new Date().toString();
-  await Notification.updateOne({
-    _id: notificationId
-  }, (error, reply) => {
-    if (error) {
-      res.status(500).json({
-        error: {
-          message: error,
-        }
-      });
-    } else {
-      res.status(201).json({
-        success: {
-          message: "Notification updated",
-        }
-      });
+  await Notification.updateOne(
```

Comment:
If you've already used find to get the notification, you can just update directly and save it, you don't need to use updateOne